### PR TITLE
Add allowdisabledfocus attribute in button components (#379)

### PR DESCRIPTION
**Pull request by SomayeEbrahimi**

### CODE REVIEW

File: src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitIconButton.razor.cs [hunk 1]
```diff
@@ -22,5 +24,17 @@ protected virtual async Task HandleOnClick(MouseEventArgs e)
                 await OnClick.InvokeAsync(e);
             }
         }
+
+        private Dictionary<string, object> SetNewAttributes()
```
Comment (ysmoradi) [inline src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitIconButton.razor.cs:28]: Is the approach is well enough? @msynk

Comment (msynk) [reply src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitIconButton.razor.cs:28]: unfortunately no :)

**Review by msynk — COMMENTED** [review]
this PR is not approved because:
- too many changes in 1 PR (it's better to split this PR into multiple PRs based on each component's changes)
- the approach selected to implement this feature is not correct

**Comment by msynk** [comment]
because there are some big issues introducing in this PR, I'm closing it.
please apply the appropriate improvements in both the issue and the PR.
@SomayeEbrahimi

**Pull request closed without merging**